test/e2e/page-objects/pages/dialog.ts

```diff
@@ -35,6 +41,10 @@ class Dialog {
     await this.driver.waitForSelector(this.setApprovalForAllSubHeadingElement);
   }
 
+  public async check_revokeSetApprovalForAllTitle() {
```

Comment:
As mentioned in the other PR, ideally we'd modularise these helpers and this could go in a `RevokeConfirmation` or at least an `ApprovalConfirmation` class?